service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDataGeneratorHelper.java

```diff
@@ -56,6 +58,8 @@
 import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.HIS_HONOUR_JUDGE;
 import static uk.gov.hmcts.reform.fpl.enums.OtherPartiesDirectionAssignee.OTHER_1;
 import static uk.gov.hmcts.reform.fpl.enums.ParentsAndRespondentsDirectionAssignee.RESPONDENT_1;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
 
 public class CaseDataGeneratorHelper {
```

Comment:
not really part of your PR, but if this class is used only in `test` and `integrationTest` modules, it shouldn't live inside `main`

Reply:
This class is not only for tests. It is used in production code

Reply:
Is it? Couldn't find reference for CaseDataGeneratorHelper 🤔 
only 17 tests :D

Reply:
it was misunderstanding, thought it is about ElementUtils and not CaseDataGeneratorHelper